### ISSUE TITLE
Putting URLSearchParams into request

### ISSUE THREAD

**Issue by MrKafr** [codemanki/cloudscraper#322]
hello, 
i am trying to scrap the website where the Cloudflare is inplemented. Before the implementation i called it like this

```js    
     const url = "website";
     form = new URLSearchParams(`items=${itemsId}&scrap=${itemsValue}&for_user=0&csrf=${csrf}`);
     return fetch(url, {
          method: "POST",
          body: form,
          headers: {
               cookie: config.cookies
          }
     })
          .catch(error => { errorHandling(error) })
          .then(res => res.json())
          .then(body => {
     })
}
```
And this works. Now, after the implementation of Cloudflare i try to use this 

```js
var options = {
          uri: "website",
          formData: { items: itemsId, scrap: itemsValue, for_user: 0, csrf: csrf },
          headers: { cookie: config.cookies }
     };
     cloudscraper.post(options)
          .catch(error => { errorHandling(error) })
          .then(body => {
          })
}
```
but it looks like the itemsId is parsed wrongly and it doesnt work. When i do the console.log of the formdata cloudscraper solution i get this

```js
formData:
   { items: '%5B%228483450990%22%5D',
     scrap: 557,
     for_user: 0,
     csrf:
      'csfr' },
```

the webside needs to get data like this.
items=%5B%228483450990%22%5D&scrap=557&for_user=0&csrf=csfr
<details><summary>Screen</summary>
https://imgur.com/a/KtVaJ5o
</details>

  * What version of Cloudscraper are you using?
    - ^4.5.0
  * What version of Node.js are you using? (Please share the process information)
    - v10.16.0
  * When and how did the problem start occurring?
   - when i started using Cloudflare instead of Fetch

- [X] I have read the [README](https://github.com/codemanki/cloudscraper#readme). (Code [examples](https://github.com/codemanki/cloudscraper/tree/master/docs/examples#readme))

**Comment by MrKafr** [comment]
I found the issue
https://github.com/codemanki/cloudscraper/issues/319
Iam thinking if this isnt a same problem... maybe the raw formData parse it with wrong &... Can i somehow console.log raw request data the cloudscrap send?

**Comment by KostyaChoporov** [comment]
Hotfixed with just replace each ``&amp;`` to ``&`` in index.js:376

**Comment by MrKafr** [comment]
Can you please give me more information how to fix it?

**Comment by codemanki** [comment]
Should be fixed in `4.6.0`. Please update and try again
